test/model/tpp/test_tpp_distribution_shape.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from typing import List, Tuple, Union
+
+# Third-party imports
+import pytest
+from mxnet import nd
+
+# First-party imports
+from gluonts.model.common import Tensor
+from gluonts.model.tpp.distribution import (
+    TPPDistributionOutput,
+    LoglogisticOutput,
+    WeibullOutput,
+)
+
+
+@pytest.mark.parametrize(
+    "distr_out, data, scale, expected_batch_shape, expected_event_shape",
+    [
+        (
+            LoglogisticOutput(),
+            nd.random.normal(shape=(3, 4, 5, 6)),
+            [None, nd.ones(shape=(1,)), nd.ones(shape=(3, 4, 5))],
+            (3, 4, 5),
+            (),
+        ),
+        (
+            WeibullOutput(),
+            nd.random.gamma(shape=(3, 4, 5, 6)),
+            # [None, nd.array([5.0])],
+            [None, nd.ones(shape=(1,)), nd.ones(shape=(3, 4, 5))],
+            (3, 4, 5),
+            (),
+        ),
+    ],
+)
+def test_distribution_output_shapes(
```

Comment:
I understand that the organization of test modules tries to mimic the one of source files, but if there is so much overlap with test/distribution/test_distribution_shapes.py, then maybe we can add these test instances to that script?

Reply:
I have moved all the tests related to `TPPDistribution` into the existing tests in `test/distribution`